sympy/polys/numberfields/galois_resolvents.py

```diff
@@ -0,0 +1,667 @@
+        Permutation(4)(1, 4)
+    ]
+
+    R6, X6 = xring("X0,X1,X2,X3,X4,X5", ZZ, lex)
+    X = X6
+
+    # First resolvent used in `_galois_group_degree_6()`:
+    H = PGL2F5()
+    term0 = X[0]**2*X[5]**2*(X[1]*X[4] + X[2]*X[3])
+    terms = set(term0.compose(list(zip(X, s(X)))) for s in H.elements)
+    F61 = sum(terms)
+    s61 = [Permutation(5)] + [Permutation(5)(0, n) for n in range(1, 6)]
+
+    # Second resolvent used in `_galois_group_degree_6()`:
+    F62 = X[0]*X[1]*X[2] + X[3]*X[4]*X[5]
+    s62 = [Permutation(5)] + [
+        Permutation(5)(i, j + 3) for i in range(3) for j in range(3)
+    ]
+
+    return {
+        (4, 0): (F40, X4, s40),
+        (4, 1): (F41, X4, s41),
+        (5, 1): (F51, X5, s51),
+        (6, 1): (F61, X6, s61),
+        (6, 2): (F62, X6, s62),
+    }
+
+
+def generate_lambda_lookup(verbose=False, trial_run=False):
+    """
+    Generate the whole lookup table of coeff lambdas, for all resolvents.
+    """
+    jobs = define_resolvents()
+    lambda_lists = {}
+    total_time = 0
+    time_for_61 = 0
+    time_for_61_last = 0
+    for k, (F, X, s) in jobs.items():
+        symmetrized, times = sparse_symmetrize_resolvent_coeffs(F, X, s, verbose=verbose)
+
+        total_time += sum(times)
+        if k == (6, 1):
+            time_for_61 = sum(times)
+            time_for_61_last = times[-1]
+
+        sv = s_vars(len(X))
+        head = f'lambda {", ".join(str(v) for v in sv)}:'
+        lambda_lists[k] = ',\n        '.join([
+            f'{head} ({wrap(f)})'
+            for f in symmetrized
+        ])
+
+        if trial_run:
+            break
+
+    table = (
+         "# This table was generated by a call to\n"
+         "# `sympy.polys.numberfields.galois_resolvents.generate_lambda_lookup()`.\n"
+        f"# The entire job took {total_time:.2f}s.\n"
+        f"# Of this, Case (6, 1) took {time_for_61:.2f}s.\n"
+        f"# The final polynomial of Case (6, 1) alone took {time_for_61_last:.2f}s.\n"
+         "resolvent_coeff_lambdas = {\n")
+
+    for k, L in lambda_lists.items():
+        table += f"    {k}: [\n"
+        table +=  "        " + L + '\n'
+        table +=  "    ],\n"
+    table += "}\n"
+    return table
+
+
+def get_resolvent_by_lookup(T, number):
+    """
+    Use the lookup table, to return a resolvent (as dup) for a given
+    polynomial *T*.
+
+    Parameters
+    ==========
+
+    T : Poly
+        The polynomial whose resolvent is needed
+
+    number : int
+        For some degrees, there are multiple resolvents.
+        Use this to indicate which one you want.
+
+    Returns
+    =======
+
+    dup
+
+    """
+    degree = T.degree()
+    L = resolvent_coeff_lambdas[(degree, number)]
+    T_coeffs = T.rep.rep[1:]
+    return [ZZ(1)] + [c(*T_coeffs) for c in L]
+
+
+if __name__ == "__main__":
+    import sys
+    verbose = '-v' in sys.argv[1:]
+    trial_run = '-t' in sys.argv[1:]
+    table = generate_lambda_lookup(verbose=verbose, trial_run=trial_run)
+    print(table)
```

Comment:
This debug script should be removed.

Reply:
Ah, for debug scripts I would definitely agree, but this is actually functional! This is so that

    (.venv) $ python -m sympy.polys.numberfields.galois_resolvents

is a reproducable way to generate the `resolvent_lookup.py` module. How about if I add a comment to that effect, right above this.

Reply:
I've added the comment.